config/webpacker.yml

```diff
@@ -41,7 +41,7 @@ development:
 
 test:
   <<: *default
-  compile: true
+  compile: false
```

Comment:
So when developing you will need to pre-compile the assets everytime? Or does webpacker-dev-server do it for you? Would be good to add some instructions here or probably use some environment. to control this.

Reply:
on development the assets will be compiled when they are required. You could also use webpacker-dev-server for autoloading and other debugging goodness

Reply:
But this state that they will not be compiled when you test locally

Reply:
not the tests, sadly, you'll need to precompile them manually.

Reply:
That is a no go without instructions. But can't we have this with environment switch?

```
compile: <%=  ENV.fetch('WEBPACKER_COMPILE', true) %>
```

Then in the CI build set this environment to `false`

Reply:
Answering: https://github.com/rails/webpacker/issues/956

So no :/